save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginConfig.kt

```diff
@@ -113,7 +113,7 @@ data class GeneralConfig(
             excludedTests ?: emptyList(),
             expectedWarningsPattern ?: defaultExpectedWarningPattern,
             runConfigPattern ?: defaultRunConfigPattern,
-        )
+        ).also { it.configLocation = this.configLocation }
```

Comment:
Previously, in stage of merging we lose the actual `configLocation` here, and used default value, because of new instance creation